simplq/pom.xml

```diff
@@ -1,135 +1,181 @@
-      <artifactId>h2</artifactId>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.postgresql</groupId>
-      <artifactId>postgresql</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.hibernate</groupId>
-      <artifactId>hibernate-core</artifactId>
-      <version>5.4.15.Final</version>
-    </dependency>
-    <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>aws-java-sdk-secretsmanager</artifactId>
-      <version>1.11.355</version>
-    </dependency>
-    <dependency>
-      <groupId>org.projectlombok</groupId>
-      <artifactId>lombok</artifactId>
-      <version>1.18.12</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-validation</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.api-client</groupId>
-      <artifactId>google-api-client</artifactId>
-      <version>1.30.11</version>
-    </dependency>
-    <dependency>
-      <groupId>com.auth0</groupId>
-      <artifactId>java-jwt</artifactId>
-      <version>3.12.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.auth0</groupId>
-      <artifactId>jwks-rsa</artifactId>
-      <version>0.9.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.google.firebase</groupId>
-      <artifactId>firebase-admin</artifactId>
-      <version>7.1.0</version>
-    </dependency>
-    
-    <!-- Tests -->
-    <dependency>
-        <groupId>org.junit.jupiter</groupId>
-        <artifactId>junit-jupiter</artifactId>
-        <version>${junitVersion}</version>
-        <scope>test</scope>
-    </dependency>
-    <dependency>
-        <groupId>org.mockito</groupId>
-        <artifactId>mockito-junit-jupiter</artifactId>
-        <version>${mockitoVersion}</version>
-        <scope>test</scope>
-    </dependency>
-    <dependency>
-        <groupId>org.mockito</groupId>
-        <artifactId>mockito-core</artifactId>
-        <version>${mockitoVersion}</version>
-        <scope>test</scope>
-    </dependency>
-    <dependency>
-        <groupId>org.assertj</groupId>
-        <artifactId>assertj-core</artifactId>
-        <version>3.19.0</version>
-        <scope>test</scope>
-    </dependency>
-  </dependencies>
+	<profiles>
+		<profile>
+			<id>dev</id>
+			<properties>
+				<liquibase.username>dev</liquibase.username>
+				<liquibase.password>password</liquibase.password>
+				<liquibase.url>jdbc:postgresql://localhost:5432/simplq</liquibase.url>
+			</properties>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+		</profile>
+		<profile>
+			<id>prod</id>
+			<properties>
+				<liquibase.username>${DB_USERNAME}</liquibase.username>
+				<liquibase.password>${DB_PASSWORD}</liquibase.password>
+				<liquibase.url>jdbc:postgresql://${DB_URL}</liquibase.url>
+			</properties>
+		</profile>
+	</profiles>
```

Comment:
To keep things uniform, let's default to the dev values and then use the values passed from command line if they exist. ([ref](https://stackoverflow.com/a/32620276))

Then lets take this as an opportunity to standardise and use the three variables, `DB_USERNAME`, `DB_PASSWORD` and `DB_URL`  [here](https://github.com/SimplQ/simplQ-backend/blob/master/simplq/src/main/java/me/simplq/config/DataSourceConfig.java) also.

Reply:
Wait... let me think. I can also come on gitter.

Reply:
I am thinking let's use std java property names, here and also in the code:
```
db.username
db.password
db.url
```

and then let's keep `url`'s value as `jdbc:postgresql://localhost:5432/simplq` (include the prefix `jdbc:postgresql://` inside the value)

Reply:
In code we can let the read the in memory db credentials from application-local.properties file, and let the URL be read from the commandline in case of other environments. 

I am guessing it should work: https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config

Reply:
@daltonfury42 in this case I can define only `db.username`, `db.password`, `db.url` with default which can be overwrite using `-D db.username`. Also this works if we use postresql or other SQL database. And this will be used on docker-compose part to start an e2e instance of simplq

Reply:
So I do not really care about in memory database.

Reply:
> So I do not really care about in memory database.
I see, I missed that point. 

Then we don't need to differentiate between dev and prod. Both are RDS postgres instances, whose values we will pick from env variables